src/test/java/tests/NavigationMenuTest.java

```diff
@@ -0,0 +1,30 @@
+package tests;
+
+import constants.NavigationLinksConstants;
+import constants.NewsMenuItemsConstants;
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+public class NavigationMenuTest extends BaseTest {
+
+    @DataProvider(name = "menuItemName")
+    public Object[][] menuItemName() {
+        return new Object[][]{{NewsMenuItemsConstants.PEOPLE}, {NewsMenuItemsConstants.CARS}, {NewsMenuItemsConstants.TECHNOLOGIES}, {NewsMenuItemsConstants.REAL_ESTATE}};
```

Comment:
Please do not write all the test data in one line, it's really hard to read

Reply:
fixed